src/Constants.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * This is a collection of values that are constant for this application,
+ * but typically would be configurable
+ */
+
+export const NUMBER_OF_SEARCH_RESULTS = 12;
```

Comment:
I don't think this should be in src/Constants.ts, it'll make the file slowly incohesive. (Principle of good design: components should be loosely coupled and highly cohesive). This will make the file slowly become a kitchen sink of different values from everywhere, that's why util files are also bad. I think this should go to fetch data for example.

Reply:
This kind of value should be configured, i.e. go into .env.production

But since we don't actually want to configure it, I'm not sure if that is worth it?

Reply:
I'm not 100% sure if this should be configurable. There is a whole school against making things configurable because it gives the software more freedom meaning it'll be less predictable and testable (you'll don't know if it's set 100, what would happen, it'll also implies people know what this configuration means.). According to a DevOps survey in google, 50% of their outages was due to misconfiguration (we had our fair share of incidents because of it). So I don't think this should be configurable.

This should be in the place that uses it the most, like the class that fetches it or part of interface of search result.

Reply:
:thinking:

Reply:
I think I know where to put it!